src/manifest/dbtProject.ts

```diff
@@ -552,7 +552,7 @@ export class DBTProject implements Disposable {
           "Encountered an unknown issue: " + exc + ".",
         ),
       );
-      return "Detailed error information:\n" + exc;
+      return "Exception:\n\nDetailed error information:\n" + exc;
```

Comment:
Why this change?

Reply:
Adding this because this function still returns a string after catching an exception. All exceptions have Exception: in the beginning so matching those. if string starts with that, i'm not generating summary for it.

Reply:
returning undefined if exception encountered after discussion with michiel.